### ISSUE TITLE
"How would you like to run EdgeDB" prompt with only one option confusing

### ISSUE THREAD

**Issue by msullivan** [edgedb/edgedb-cli#336]
`edgedb project init` gave me the prompt
```
Type a number to select an option:
How would you like to run EdgeDB for this project?
1. Local (native package)
>
```

I found this confusing: there was only one option (which wasn't selected as a default), and I also didn't really have a clear idea what the other (missing!) possibilities would be. (Though I could guess)

We should probably not ask this question if there is only one option, or else perhaps provide enough context to know what the other options might be and what configuration space changes would be needed for them to appear? I'm not sure.

**Comment by tailhook** [comment]
Yes this is good question. For specific case we may provide instructions for installing docker. But if docker is the only option we can't.

On the other hand:
1. I want installation method (especially if that's a docker) be explicitly known
2. We should have cloud shortly and cloud should always be available, right?

**Comment by 1st1** [comment]
> I want installation method (especially if that's a docker) be explicitly known

We can just print a different prompt then: "There's only one available method of running EdgeDB on your platform: Docker. Would you like to run EdgeDB with it?"

> We should have cloud shortly and cloud should always be available, right?

Right, but I doubt about "shortly", it can take months. So I'd optimize the UX for what we have now.
